plus/pom.xml

```diff
@@ -55,6 +55,11 @@ limitations under the License.
       <groupId>org.apache.calcite.avatica</groupId>
       <artifactId>avatica-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
```

Comment:
It does look more like a `<scope>test</scope>` dependency, doesn't it?
I would like to refrain from adding regular dependencies for test purposes.